tests/test_cmd_signature.py

```diff
@@ -4337,11 +4343,28 @@ def test_sig_manifest_7_allzip_2(runtmp):
     assert "dna-sig.noext" in filenames
 
 
-def test_sig_manifest_7_allzip_3(runtmp):
-    # the existing manifest contains 'dna-sig.noext' whther or not -f is
-    # used.
+def test_sig_manifest_7_allzip_3_no_rebuild(runtmp, cli_v4_only):
+    # the manifest contains 8 entries.
+    # note: --no-rebuild-manifest is default behavior on v4, but not on v5.
```

Comment:
This is a confusing note to me - make it seem like the default is not to rebuild the manifest (though default v4 is to rebuild). The one above is clearer:

"# note: default in v4 is to rebuild manifest"

Reply:
you are correct! thx.